src/Shared/LayoutRenderers/AspNetRequestValueLayoutRenderer.cs

```diff
@@ -206,5 +215,12 @@ private static string LookupItemValue(string key, HttpRequest httpRequest)
             return null;
         }
 #endif
+
+#if ASP_NET_CORE3
+        private static string LookupServerVariableValue(string key, HttpContext httpContext)
```

Comment:
Can this method be changed to this:

```c#
private static string LookupServerVariableValue(string key, HttpRequest httpRequest)
{
#if ASP_NET_CORE3
     return httpRequest.HttpContext?.TryGetFeatureCollection()?.Get<IServerVariablesFeature>()?[key];
#else
     return nulll;
#endif
}
```

And be placed together with the other `ASP_NET_CORE`-specific ones.

Reply:
Complete